_docs/aws_resources.md

```diff
@@ -8,6 +8,168 @@ permalink: /aws_resources.html
+- Now, there should be two buttons: "Create Item", and "Actions"
+- Select "Actions"
+- Under "Actions", there is the option "Export to .CSV"
+- Select this option
+- A .CSV file will be automatically saved to your computer
+
+This can be done for as many, or as little table items as needed.
+```
+
+.JSON (by hand)
+```
+DynamoDB does not have inbuilt exporting of .JSON files, but it is not to difficult to get your information exported with using Data Pipeline, or Lambda. It is a bit of a brute force solution, but it is simple and functional.
+
+To do so, follow the steps listed above to export a .CSV file. Then, convert that .CSV file into .JSON. It's that easy!
+
+There are a number of sites that can do the conversion, but an example of one such sight can be found here: https://csvjson.com/csv2json
+```
+
+.JSON (Lambda)
+```python
+import json
+import boto3 #sdk
+s3_client = boto3.client('s3')
+client = boto3.client('dynamodb')
+s3 = boto3.resource("s3")
+
+# BOTO 3 DOCS: https://boto3.amazonaws.com/v1/documentation/api/latest/index.html
+# Python vers 3.6
+
+def lambda_handler(event, context):
+    # gets all info from table you want to export
+    response = client.scan(
+    TableName='FactsImport',
+    )
+
+    # the s3 bucket you want to upload into
+    bucket_name = "crobfactsample"
+    file_name = "info1.json"
+    # used for making your file to upload to s3
+    lambda_path = "/tmp/" + file_name
+    # used for where your file will be uploaded within s3 bucket
+    # stuff before '/' will be path
+    # will create new path if none there before
+    s3_path = "crowTest/" + file_name
+    # handles writing the 'dict' data type to .json file
+    # good for keeping formatting
+    with open(lambda_path, 'w') as f:
+        json.dump(response, f, sort_keys = True)       
+    # uploads file from lambda_path to your s3_path, in your s3 bucket
+    s3.Bucket(bucket_name).upload_file(lambda_path, s3_path)
+```
+
+
+## Reserved Words
+
+When working with tables in AWS, it's important to keep mind of a few words that should not be used. These are words in use by the database itself for a variety of different reasons, and using these terms as a primary key, or sort key, will cause errors when trying to call those columns from a JavaScript call.
+
+A few example words are:
+- LOCATION
+- ATTRIBUTE
+- BLOB
+- CLASS
+- DIAGNOSTICS
+- GROUP
+
+**DO NOT USE THESE WORDS**
+
+They will only cause problems for you down the line!
+
+Before finalizing your database schema, be sure to check ALL TERMS against these reserved words. If you use on of these terms for your partition key, or sort key, there will be no way to change them. Failure to double check may mean your group will have to terminate your table, and recreate it so it no longer uses these terms.
+
+For a full list of terms, check out the resource for DynamoDB: [Reserved Words in DynamoDB
+PDF](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html)
```

Comment:
Removing the PDF part since this is an HTML document
```suggestion
For a full list of terms, check out the resource for DynamoDB: [Reserved Words in DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html)
```